regen-network/regen-web
# fix: 1116 silent error on sell order creation

## Description

Closes: regen-network/regen-registry#1116

- fix silent error on failed create sell order transaction
- simplify error modal conditional display
- change error button label on error modal for failed create sell order transaction
- fix console error on incorrect SVG properties
- fix console error on nested `p` tag

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...


#### Setting up backport PRs

After merging your PR to `master`, set up backports by doing the following:

1. If your branch does not have merge commits, add the following comment to
   your PR, `@Mergifyio backport redwood hambach`.

### How to test

1. Go to https://deploy-preview-1132--regen-registry.netlify.app/
2. Try to. create a sell order
3. The error popup should be displayed because the preview is still targeting v4 testnet

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
